spec/widgets/time-series/torque-histogram-view.spec.js

```diff
@@ -32,7 +32,7 @@ describe('widgets/time-series/torque-histogram-view', function () {
       start: 0,
       end: 1000
     }, {
-      vis: vis
+      engine: vis._engine
```

Comment:
Should we expose a `getEngine` method?

Reply:
This is required only in some tests, and it could change in the future.